yarn.lock

```diff
@@ -8326,7 +8669,7 @@ ethereum-common@^0.0.18:
   resolved "https://registry.yarnpkg.com/ethereum-common/-/ethereum-common-0.0.18.tgz#2fdc3576f232903358976eb39da783213ff9523f"
   integrity sha1-L9w1dvIykDNYl26znaeDIT/5Uj8=
 
-ethereum-cryptography@^0.1.2, ethereum-cryptography@^0.1.3:
+ethereum-cryptography@0.1.3, ethereum-cryptography@^0.1.2, ethereum-cryptography@^0.1.3:
```

Comment:
Weird double-up on ethereum-cryptography@0.1.3 here. One is pegged to the exact 0.1.3, the other is pegged to the minor.

Probably nothing to do - just a shame the tools do this :(

Reply:
Yeah I just tried cleaning up the yarn.lock with this tool: https://www.npmjs.com/package/yarn-deduplicate

Seems useful and added a bunch of edits in this commit https://github.com/across-protocol/sdk-v2/pull/117/commits/7949b87c429c8b0333d7f7740d545a68c1c1997c

Reply:
This tool is also very useful https://www.npmjs.com/package/yarn-check